ml-conversational-analytic-tool/__init__.py

```diff
@@ -0,0 +1 @@
+import os, sys; sys.path.append(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
I'm not clear why the system path is being updated here.  But also, the last line of a file should end with a newline. And use one import per module to simplify the import section.

Reply:
this is required to make the modules(commentAnalysis.py in this case) visible to the testing modules. There are a couple of ways to do it and this seems to work.